jwst/linearity/linearity.py

```diff
@@ -11,7 +11,22 @@
 
 
 def do_correction(output_model, lin_model):
-
+    """
+    Apply the linearity correction to the data.
+
+    Parameters
+    ----------
+    output_model : `~jwst.datamodel.RampModel`
+        Science data to be corrected.
+
+    lin_model : `~jwst.datamodel.LinearityModel`
+        Linearity reference file model.
+
+    Returns
+    -------
+    output_model : `~jwst.datamodel.RampModel`
+        Linearity corrected science data.
```

Comment:
Oops, I just realized there's one more typo to fix here -- it should be "datamodels", not "datamodel".  Or else just specify "RampModel" for all three of these.

```suggestion
    output_model : `~jwst.datamodels.RampModel`
        Science data to be corrected.

    lin_model : `~jwst.datamodels.LinearityModel`
        Linearity reference file model.

    Returns
    -------
    output_model : `~jwst.datamodels.RampModel`
        Linearity corrected science data.
```